plugin/frontend/page.php

```diff
@@ -241,18 +241,20 @@ function images( $client, $dir, $options, $skip, $remaining ) {
 			'pageSize'              => 1000,
 		];
 		if ( $options->get_by( 'image_ordering' ) === 'time' ) {
-			$params['fields'] = 'nextPageToken, files(id, thumbnailLink, createdTime, imageMediaMetadata(time))';
+			$params['fields'] = 'nextPageToken, files(id, thumbnailLink, createdTime, imageMediaMetadata(time), description)';
 		} else {
 			$params['orderBy'] = $options->get( 'image_ordering' );
-			$params['fields']  = 'nextPageToken, files(id, thumbnailLink)';
+			$params['fields']  = 'nextPageToken, files(id, thumbnailLink, description)';
 		}
 		$response = $client->files->listFiles( $params );
 		if ( $response instanceof \Sgdg\Vendor\Google_Service_Exception ) {
 			throw $response;
 		}
 		foreach ( $response->getFiles() as $file ) {
+            $description = $file->getDescription();
 			$val = [
 				'id'        => $file->getId(),
+                'description' => ($description === NULL ? '' : $description),
```

Comment:
This could be done *nicer*...

Reply:
https://developer.wordpress.org/reference/functions/esc_attr/

Reply:
1) I will fix the whitespace in page.php
2) What kind of _nicer_ do you want?
`$description = $file->getDescription();
if ($description === NULL) {
$description = '';
}`
Or some other way? Is the ternary operator the thing you don't like?
3) ok, thanks, will use the esc_attr function

Reply:
Hi, sorry, I should've been more concrete.
1. Thanks
2. Ternary is not the problem - I was thinking more along the lines of the null check - is that really the value returned by the library? Shouldn't it be something like `isset()` or the like? Also, I was thinking of the null coalesce operator but didn't realize that that isn't in PHP 5.6 so we can't use it...
3. Thanks

Reply:
2. I believe it really returns NULL (otherwise this check wouldn't work?), I am not a php programmer and this $var === NULL is what I have found as recommended via googling.  Should I do it in some different way?